src/layout.rs

```diff
@@ -337,6 +340,7 @@ impl Default for Layout {
 }
 
 impl Layout {
+    const DEFAULT_CACHE_SIZE: usize = 16;
```

Comment:
To make this appear in the rendered docs:
```suggestion
    pub const DEFAULT_CACHE_SIZE: usize = 16;
```
(you can run `cargo doc --open` to view the docs locally)